nilearn/datasets/tests/test_func.py

```diff
@@ -350,184 +408,156 @@ def test_check_parameters_megatrawls_datasets(request_mocker):
 
     for invalid_input_dim in [1, 5, 30]:
         with pytest.raises(
-                ValueError,
-                match=message.format('dimensionality', invalid_input_dim)):
+            ValueError,
+            match=message.format("dimensionality", invalid_input_dim),
+        ):
             func.fetch_megatrawls_netmats(dimensionality=invalid_input_dim)
 
-    for invalid_input_timeserie in ['asdf', 'time', 'st2']:
+    for invalid_input_timeserie in ["asdf", "time", "st2"]:
         with pytest.raises(
-                ValueError,
-                match=message.format('timeseries', invalid_input_timeserie)):
+            ValueError,
+            match=message.format("timeseries", invalid_input_timeserie),
+        ):
             func.fetch_megatrawls_netmats(timeseries=invalid_input_timeserie)
 
-    for invalid_output_name in ['net1', 'net2']:
+    for invalid_output_name in ["net1", "net2"]:
         with pytest.raises(
-                ValueError,
-                match=message.format('matrices', invalid_output_name)):
+            ValueError, match=message.format("matrices", invalid_output_name)
+        ):
             func.fetch_megatrawls_netmats(matrices=invalid_output_name)
 
 
 def test_fetch_megatrawls_netmats(tmp_path, request_mocker):
     # smoke test to see that files are fetched and read properly
     # since we are loading data present in it
-    files_dir = str(tmp_path / 'Megatrawls'
-                    / '3T_Q1-Q6related468_MSMsulc_d100_ts3')
+    files_dir = str(
+        tmp_path / "Megatrawls" / "3T_Q1-Q6related468_MSMsulc_d100_ts3"
+    )
```

Comment:
unrelated to formatting, but we don't need to use `os.makedirs` or `os.path.join`, we can rely on Pathlib  (`files_dir.mkdir(parents=True)`, `files_dir / "Znet2.txt"`....)